docs/input-variables.md

```diff
@@ -112,6 +112,7 @@ The following input variables are used to configure the inbound security rules o
 name                                | default                 | description
 ------------------------------------|-------------------------|------------
 network_cidrs                       | See map in variables.tf | A CIDR notation IP range of the VCN and its subnets.
+network_subnet_dns                  | See map in variables.tf | A DNS label for each of the subnet in the VCN (Max 15 characters)
```

Comment:
A DNS label for each of the subnet in the VCN => A DNS label for each of the subnets in the VCN